diff --git a/BUILD.gn b/BUILD.gn
index cd90de0e82..45ee443ea6 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -73,6 +73,12 @@ config("skia_public") {
     defines += [ "_LIBCPP_ENABLE_ASSERTIONS=1" ]
   }
 
+  # The renderer wants chromium linux-like text, copied from https://git.io/JUMir#L144-L147
+  defines += [
+    "SK_GAMMA_EXPONENT=1.2",
+    "SK_GAMMA_CONTRAST=0.2",
+  ]
+
   # Some older versions of the Clang toolchain change the visibility of
   # symbols decorated with API_AVAILABLE macro to be visible. Users of such
   # toolchains suppress the use of this macro till toolchain updates are made.
